common/cache_policy/lru_policy.go

```diff
@@ -50,11 +44,30 @@ func (cache *LRUCache) Get(bk int64) (*CacheBlock, bool) {
 	return cb, found
 }
 
-//Put: Inserts the key,value pair in LRUCache.
-func (cache *LRUCache) Put(key int64, value *CacheBlock) {
+// return true if eviction happened, return false otherwise
+func (cache *LRUCache) findCleanBlockToEvict() bool {
+	node := cache.List.Back()
+	pair := getKeyPair(node)
+	for i := 0; i < cache.List.Len(); i++ {
+		if !pair.value.Dirty {
+			fmt.Println("somehow not dirty")
+			cache.Remove(pair.key)
+			return false
+		} else {
+			node = node.Next()
```

Comment:
We started with cache.list.Back() (last node in list) so for iteration we shall be going node.Prev() and not .Next() right ?